blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -22,9 +22,14 @@ export class RunDetailsPipeline extends Component {
     }
     componentDidMount() {
         const { result } = this.props;
-        if (!result.isQueued()) {
-            document.addEventListener('wheel', this._onScrollHandler, false);
-            document.addEventListener('keydown', this._handleKeys, false);
+        if (!result.isCompleted()) {
+            document.addEventListener('scroll', this._onScrollHandler, true);
+            const thisEl = ReactDOM.findDOMNode(this);
+            const fullscreenContents = thisEl.parentElement.parentElement; // this is the actual div that scrolls
```

Comment:
@sophistifunk A quick note here, this bit fixes the scrolling by basically restoring the browser's natural scrolling by focusing on the content within the dialog, it would also fix tab order. Probably this behavior should be pushed to the dialog or something more generically (but then often you want to focus on the first input or something specific). The problem was it was focused on the outer page so arrow keys didn't work, scroll had odd behavior.